aztk/models/cluster_state.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class ClusterState(Enum):
+    deleting = "deleting"
+    resizing = "resizing"
+    steady = "steady"
+    stopping = "stopping"
```

Comment:
`stopping` feels weird actually in this enum, could it be renamed to stoppingResize?

Reply:
Sure. Camel casing the Enum looks weird to me, though. Looking back at python docs, it seems like `Enum.SOME_NAME` is the idiomatic casing. I guess I should align to that for all.

Reply:
Actually, I'm going to stick with `Enum.some_value` since that is what the Batch SDK does.